### ISSUE TITLE
Nextcloud AIO Eats All My Memory

### ISSUE THREAD

**Issue by Lwfrancisco** [nextcloud/all-in-one#906]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Update Nextcloud AIO to latest
2. Unsure what causes it exactly, but I've now caused it by syncing via the Windows Desktop app and again while deleting 3 files in the webui.
3. Notice RAM usage spikes to maximum on the system.

### Expected behavior 
Delete and sync files.

### Actual behavior 
System freezes or slows down terribly as RAM usage spikes. This LXC container ONLY has nextcloud-aio on it.

### Host OS 
Proxmox LXC - Debian

#### Nextcloud AIO version 
Latest

#### Current channel 
Stable

#### Other valuable info 
The host system is not by any means overloaded. Neither is the Container. Started on 4GB RAM, but encountered this issue. Upgraded container to 6GB RAM and again encountered this issue shortly after.

This is a low volume server, with only a handful of users and in all likelihood only 1 concurrent user throughout.

The first time, when encountering during a "sync", I was able to open top and see that php and postgres had spawned a ton of processes. I don't know if this is particularly normal but there seemed to be quite a few processes. The second time, now with 6GB of RAM, the system just totally froze up and could not run top after half an hour.

Please let me know what logs I need to provide!

Screenshot of both instances (also shows approximately how long it took after the system was again up and running for this to happen once more).
![ProxmoxNextcloudAIOCapture](https://user-images.githubusercontent.com/8664661/178095372-aba4651c-b50a-4fa2-bf39-35d7a3168b1b.PNG)

**Comment by szaimen** [comment]
Hm... Are you using external storage in your Nextcloud?

Also how many desktop clients are set up to sync files? And does the sync finish successfully?

**Comment by Lwfrancisco** [comment]
Nope, no external storage app. Also, likely only one desktop sync is set up. Mayhaps 2 phone syncs.

The sync will most often finish. I've only encountered this weird behavior a few times and no consistent usage pattern. Is there a log I can grab that would help clarify this?

**Comment by johnnyq** [comment]
More than likely this is just cached memory and is completely normal. Cached memory will be released when needed. Are you noticing any slowdowns after it fills up? Run command free -m from the next AIO host and paste results.

**Comment by szaimen** [comment]
I fear I don't have any clue what could trigger this....

**Comment by szaimen** [comment]
I'll move this to discussions for now as I cannot reproduce this...
